src/main/resources/application.conf

```diff
@@ -0,0 +1,7 @@
+application {
+  masterUrl:"local[*]"
+  save_path = "twitter-analytics/dataLake/eng-tweets"
+  date_format = "YYYYMMdd HH.mm"
```

Comment:
is it a dynamic parameter?

Reply:
or should it be?

Reply:
Same goes for date_format. It was put in conf file, just incase we need to change the pattern of Date. So if this is going to be fixed always, will remove it from here.